src/Core/SettingsChangesHistory.cpp

```diff
@@ -88,6 +88,7 @@ static std::initializer_list<std::pair<ClickHouseVersion, SettingsChangesHistory
             {"min_free_disk_bytes_to_perform_insert", 0, 0, "New setting."},
             {"min_free_disk_ratio_to_perform_insert", 0.0, 0.0, "New setting."},
             {"enable_named_columns_in_function_tuple", false, true, "Re-enable the setting since all known bugs are fixed"},
+            {"strict_named_tuple_conversion", true, true, "New setting"},
```

Comment:
Should be moved to 24.11 and be `false, true`. This also makes the PR an improvement and not a breaking change